fuse_models/include/fuse_models/common/sensor_proc.h

```diff
@@ -737,6 +739,7 @@ inline bool processDifferentialPoseWithTwistCovariance(
   const geometry_msgs::PoseWithCovarianceStamped& pose2,
   const geometry_msgs::TwistWithCovarianceStamped& twist,
   const fuse_core::Matrix3d& minimum_pose_relative_covariance,
+  const fuse_core::Matrix3d& minimum_twist_covariance,
```

Comment:
My only concern here is similarity in name to "minimum_pose_relative_covariance". The pose covariance is added to the output, but the twist covariance is subtracted. For people like me that don't read the manual thoroughly, can we change the name to .... "twist_covariance_offset" ... or anything else really? Just a name that is not parallel to "minimum_pose_relative_covariance", to emphasize the difference in functionality.

Reply:
I agree. I admit I also had the feeling it has a name very similar to `minimum_pose_relative_covariance` and could lead to confusion. I'm going to rename it to `twist_covariance_offset_diagonal`; I can't think of a better name and I'm just prefixing it with `_diagonal` for consistency with other _similar_ parameters.

Reply:
Done.

Finally using `twist_covariance_offset_diagonal` for the parameter and `twist_covariance_offset` for the variable/attribute.

I also added the word `offset` to the comment above. The one you suggested a change for.